src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -28,6 +28,12 @@
         </service>
 
         <service id="translator" class="Symfony\Component\Translation\IdentityTranslator">
+            <argument type="service" id="translator.formatter" />
+        </service>
+
+        <service id="translator.formatter" alias="translator.formatter.default" />
+
+        <service id="translator.formatter.default" class="Symfony\Component\Translation\Formatter\TransMessageFormatter" public="false">
```

Comment:
I would not name the service ``.default`` but ``.trans_message``, to match the name of the class (and make it easier to change the default later, if we don't have to rename services for that, as the actual default is handled by the alias, not by the service name)